frontend/admin/.babelrc

```diff
@@ -2,6 +2,13 @@
   "plugins": [
     "@babel/plugin-proposal-class-properties",
     "@babel/plugin-proposal-private-methods"
+  ],
+  "presets": [
+    ["@babel/preset-env", {
+      "useBuiltIns":"usage",
+      "corejs":3,
+      "targets":"firefox 50, IE 11"
```

Comment:
Honestly, I think we can remove IE. We don't care about supporting it anymore. We should probably add more targets though, including a version of Chrome.

Reply:
My recommendation on targets after some [slack discussion](https://talent-cloud.slack.com/archives/CB424V61J/p1648565972627749) and reading [browserlist docs](https://github.com/browserslist/browserslist): `> 0.5% in CA, last 2 versions, Firefox ESR, not dead, not IE 11`

Reply:
fyi updated Google Analytics so we have more reliable data about this stuff in the future (which some of these tools @tristan-orourke is alluding to even "auto-manage" with a Google Analytics API key to check on browser stats directly 🙃 )

https://talent-cloud.slack.com/archives/CB424V61J/p1648581658632169?thread_ts=1648565972.627749&cid=CB424V61J

Reply:
For clarity, would recommend storing this in own config file (or [even package.json](https://github.com/compdemocracy/polis/blob/eb1656db2ebc2e348c0e85f688cb1dff9924b413/client-participation/package.json#L87-L93)) so that other tools can find it. But I don't think this is a big deal for this PR (unless other changes are being made, and it's simple to do)

Reply:
Interesting idea.  I'm worried about scope creep so I opened a new issue for it: #2360